### ISSUE TITLE
About the training process

### ISSUE THREAD

**Issue by lcy0604** [Niwhskal/SRNet#5]
Hello, I really appreciate the your re-implement for text editing. I follow your instruction to run the project and only changes the batch size to 4. I found the loss converge really slow and the results are bad. After 20w iters, the G loss is 19, D_fus loss is 4.8, D_bg loss is 5.5071. Is it normal or some problems exist?

**Comment by Niwhskal** [comment]
Hello lcy0604,

There were minor bugs in the model. I have fixed all of them. The model now works as intended. Pull all changes to try it out. Pre-trained weights have also been updated. Refer the readme file for more information.

Thank you.
